test/unit/proxy_api/dhcp_test.rb

```diff
@@ -15,6 +15,41 @@ def setup
     assert_equal(expected, @dhcp.url)
   end
 
+  context '#record' do
+    test 'retrieves an array with a single dhcp record' do
+      fake_response = {'name'=>'www.example.com', 'mac'=>'00:11:22:33:44:55', 'ip'=>'192.168.0.10', 'filename'=>'pxelinux.0', 'nextServer'=>'1.2.3.4', 'hostname'=>'www.example.com', 'subnet'=>'192.168.0.0/255.255.255.0'}
+      @dhcp.expects(:get).with('192.168.0.0/mac/192.168.0.10').returns(fake_rest_client_response(fake_response))
+      result = @dhcp.record('192.168.0.0', '192.168.0.10')
+      assert_kind_of Net::DHCP::Record, result
+      assert_equal({:hostname=>"www.example.com",
+                    :mac=>"00:11:22:33:44:55",
+                    :ip=>"192.168.0.10",
+                    :network=>"192.168.0.0",
+                    :nextServer=>"1.2.3.4",
+                    :filename=>"pxelinux.0",
+                    :name=>"www.example.com",
+                    :related_macs=>[]}, result.attrs)
+    end
+  end
+
+  context '#records_by_ip' do
+    test 'retrieves an array with a single dhcp record' do
+      fake_response = [{'name'=>'www.example.com', 'mac'=>'00:11:22:33:44:55', 'ip'=>'192.168.0.10', 'filename'=>'pxelinux.0', 'nextServer'=>'1.2.3.4', 'hostname'=>'www.example.com', 'subnet'=>'192.168.0.0/255.255.255.0'}]
```

Comment:
Looks like you can extract the first 2 lines of these tests to a method or put them in the same context. I believe these two tests `context` should actually be `describe` instead, it's most commonly used to describe a method, whereas `context` is for situations.